x/wasm/keeper/snapshotter.go

```diff
@@ -87,7 +87,7 @@ func (ws *WasmSnapshotter) Snapshot(height uint64, protoWriter protoio.Writer) e
 
 		return false
 	})
-
+	cacheMS.Close()
```

Comment:
should we `defer` call this after creating the store for export?

Reply:
ah good point!